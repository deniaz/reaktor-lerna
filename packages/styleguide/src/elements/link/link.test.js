import React from 'react';
import { create } from 'react-test-renderer';

import { Link } from '.';

describe('Element', () => {
  describe('Link', () => {
    it('matches snapshot', () => {
      const fn = jest.fn();
      const component = create(<Link href="#">A Link</Link>);
      const tree = component.toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
