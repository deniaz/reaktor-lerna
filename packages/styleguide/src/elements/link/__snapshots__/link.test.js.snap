// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Element Link matches snapshot 1`] = `
<a
  className="link"
  href="#"
>
  A Link
</a>
`;
